docs/mine/lotus/disputer.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Disputer
+description: Verifying submitted Window PoSts is expensive, and that cost can drastically increase when network congestion causes the gas base fee to rise. To address this, Filecoin Improvement Proposal allows miners to optimistically accept Window PoSts on-chain without verification, allowing them to be disputed later by off-chain-verifiers.
```

Comment:
- Specifically, FIP-0010
- It doesn't so much "allow" for optimistic acceptance, it makes it the behaviour for everyone
- The change isn't for miners to start optimistically accepting PoSts, it's for all node operators

Reply:
```suggestion
description: Verifying submitted Window PoSts is expensive, and that cost can drastically increase when network congestion causes the gas base fee to rise. To address this, Filecoin Improvement Proposal FIP-0010 enables node operators to optimistically accept Window PoSts on-chain without verification, allowing them to be disputed later by off-chain-verifiers.
```